tests/test_database.py

```diff
@@ -76,6 +76,15 @@ async def test_database_create_03(self):
         finally:
             await tb.drop_db(self.con, 'databasename')
 
+    async def test_database_create_04(self):
+        if not self.has_create_database:
+            self.skipTest("create database is not supported by the backend")
+
+        # create database name that conflicts with names in schema
+        await self.con.execute('CREATE DATABASE range;')
```

Comment:
Could we do some testing that having this name collision doesn't lead to any trouble?

Reply:
Like what? Write to the new database?

Reply:
No, like using the objects that it collided with